src/approaches/online_nsrt_learning_approach.py

```diff
@@ -45,14 +52,18 @@ def get_interaction_requests(self) -> List[InteractionRequest]:
         # explored at each timestep is a hyperparameter. The train task
         # is randomly selected.
 
-        # Create the explorer. Note that GLIB and greedy lookahead are not yet
-        # supported.
+        # Create the explorer. Note that greedy lookahead is not yet supported.
+        preds = self._get_current_predicates()
         explorer = create_explorer(CFG.explorer,
```

Comment:
so would this crash if CFG.explorer == random_options bc things like babble_predicates and atom_score_fn can't be passed as kwargs?

Reply:
no it won't crash -- that stuff is handled within `create_explorer`